conf/application-json-logger.xml

```diff
@@ -7,6 +7,8 @@
 
     <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
 
+    <logger name="application" level="${logger.application:-WARN}"/>
```

Comment:
I think this `application` logger refers to the deprecated uppercase L `play.api.Logger` - I think that by changing to use the `play.api.Logging` logger, the logging above is configurable from the root logger now instead

I could be wrong though!

Reply:
This is from the logger that is set by [the trait here](https://github.com/hmrc/transit-movements-trader-eis-router/blob/master/app/logging/Logging.scala#L23)

Reply:
Ah I see, good spot, I didn't realise it was using a custom trait rather than the Play one